geonode/upload/api/tests.py

```diff
@@ -467,15 +468,15 @@ def test_rest_uploads(self):
         self.assertEqual(upload_data['name'], 'relief_san_andres')
 
         if upload_data['state'] != enumerations.STATE_PROCESSED:
-            self.assertEqual(upload_data['progress'], 100.0)
+            self.assertGreaterEqual(upload_data['progress'], 90.0)
             self.assertIsNone(upload_data['resume_url'])
             self.assertIsNone(upload_data['delete_url'])
             self.assertIsNotNone(upload_data['detail_url'])
 
             self.assertIn('uploadfile_set', upload_data)
             self.assertEqual(len(upload_data['uploadfile_set']), 2)
         else:
-            self.assertEqual(upload_data['progress'], 100.0)
+            self.assertGreaterEqual(upload_data['progress'], 90.0)
```

Comment:
The async task takes a bit to update the percentage. The change on the view does not hide the exception anymore and therefore the upload waiting process exists early instead of stucking on the async response.